openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -585,7 +585,9 @@ def _validate_password(password, username=None, email=None):
 
 
 def _validate_confirm_password(confirm_password, password):
-    if not confirm_password or confirm_password != password:
+    if not confirm_password:
+        raise errors.AccountPasswordInvalid('Please enter your Confirm password.')
```

Comment:
Please enter your Confirm password. -> Please confirm your password. / Password confirmation is required

Reply:
fixedd